sdk/communication/azure-communication-chat/azure/communication/chat/__init__.py

```diff
@@ -56,8 +54,6 @@
     'identifier_from_raw_id',
     'PhoneNumberIdentifier',
     'PhoneNumberProperties',
-    'MicrosoftBotIdentifier',
-    'MicrosoftBotProperties',
```

Comment:
We haven't added the `getattr` method to chat?

Reply:
I reverted all beta bot identifier changes for chat SDK as they haven't released anything yet